tutorials/sphinx-tutorials/multiagent_competitive_ddpg.py

```diff
@@ -486,8 +486,8 @@
         out_keys=[(group, "action")],
         distribution_class=TanhDelta,
         distribution_kwargs={
-            "low": env.full_action_spec[group, "action"].space.low,
-            "high": env.full_action_spec[group, "action"].space.high,
+            "low": env.single_full_action_spec[group, "action"].space.low,
+            "high": env.single_full_action_spec[group, "action"].space.high,
```

Comment:
The line before:
`spec=env.full_action_spec[group, "action"],`
should also be changed then.